SU2_CFD/include/fluid/CDiffusivityModel.hpp

```diff
@@ -0,0 +1,49 @@
+/*!
+ * \file CDiffusivityModel.hpp
+ * \brief Interface class for defining mass diffusivity models.
+ * \author S. Vitale, M. Pini, G. Gori, A. Guardone, P. Colonna, T. Economon
+ * \version 7.0.6 "Blackbird"
```

Comment:
update version and copyright year please, check other files

Reply:
done